### PR TITLE
Add game class

### PR DESCRIPTION
**Pull request by keza681**
### Hello @Benmuiruri , @selma-belhadj 
In this PR, I have 
- Created a Game class in a separate .rb file.
 ----------------------------------------------------------------

### CODE REVIEW

File: database_schemas/schema.sql [hunk 1]
```diff
@@ -32,3 +44,16 @@ CREATE TABLE books (
...
+    id INT PRIMARY KEY,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL
+);
+
+CREATE TABLE games (
+    id INT PRIMARY KEY,
+    multiplayer VARCHAR(100),
+    last_played_at DATE,
+    FOREIGN KEY(item_id) REFERENCES items(id)
```
Comment (selma-belhadj) [inline database_schemas/schema.sql:58]: - [ ] I think you should create a foreign key on `author_id` that references `authors` table.

**Review by selma-belhadj — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️
Hello @keza681,
Good job so far!
✔️ Requirements are implemented well
✔️ Use of Ruby best practices
However, there are some simple issues that I would like you to add so as to make our codes maintainable

### Requested improvements ♻️
❌  check ruby offences by `rubocop --auto-correct-all `
Check the comments under the review.

Otherwise Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

File: classes/game.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require_relative './items'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+  attr_reader :id, :publish_date
+
+  def initialize(multiplayer, last_played_at, publish_date)
+    super(publish_date)
+    @id = Random.rand(1..500)
```
[the remark this answers is not in the document]
Comment (keza681) [reply classes/game.rb:9]: Thanks, @selma-belhadj for the feedback

File: classes/game.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+    @publish_date = publish_date
```
[the remark this answers is not in the document]
Comment (keza681) [reply classes/game.rb:12]: this is helpfull. thanks

File: database_schemas/schema.sql (hunk 1 above)
Comment (keza681) [reply database_schemas/schema.sql:58]: 😃 thank you @selma-belhadj for the feedback

File: database_schemas/schema.sql [hunk 4]
```diff
@@ -1,3 +1,15 @@
...
+    id SERIAL PRIMARY KEY,
+    archived BOOLEAN DEFAULT,
+    author_id SERIAL NOT NULL,
+    label_id SERIAL NOT NULL,
+    genre_id SERIAL NOT NULL,
+    publish_date TIMESTAMP NOT NULL,
+    CONSTRAINT author_fk FOREIGN KEY (author_id) REFERENCES author(id),
+    CONSTRAINT label_fk FOREIGN KEY (label_id) REFERENCES label(id),
+    CONSTRAINT genre_fk FOREIGN KEY (genre_id) REFERENCES genre(id)
+);
```
[the remark this answers is not in the document]
Comment (keza681) [reply database_schemas/schema.sql:11]: understood 🙇

**Review by selma-belhadj — APPROVED** [review]
### Hello @keza681 , 
Your project is complete! There is nothing else to say other than... it's time to merge it :
Congratulations! **🎉**

✔️ Requirements are implemented well
✔️ No linter errors
✔️ Use of Ruby best practices
✔️ Corrected some errors which were in the codes
#### However before merging, there are some simple issues that I would like you to add so as to make our codes maintainable.
Check the comments under the review.

**Pull request merged**